livre/core/data/NodeId.cpp

```diff
@@ -30,30 +30,26 @@ NodeId::NodeId( const Identifier& identifier )
     : _id( identifier )
 {}
 
-NodeId::NodeId( const uint32_t level,
-                const Vector3ui& position,
+NodeId::NodeId( const uint32_t level, const Vector3ui& position,
```

Comment:
And, I am not seeing any functional change on this file. Good to clean but does it match with keep changes small ?